Link definition of media profile in senders's behavior (#7)
AMWA-TV/nmos-sink-metadata-processing: docs/1.0. Overview.md

### Media Profile

> A Collection of formats that are acceptable for all Receivers implied to be a part of a connection. A Media Profile consist of separate parameters required to set a Sender, Flow, and/or Source.

Reviewer: ```suggestion
Description of a format that is acceptable for all Receivers implied to be a part of a connection. A Media Profile consist of separate media parameters required to set a Sender, Flow, and/or Source.
```
It is a single format, not a collection.

Author: :confused: https://github.com/AMWA-TV/nmos-sink-metadata-processing/blob/75e292aa65a28d63a057a0fded694cb0d30a47f8/APIs/schemas/media_profiles.json#L3
